Modules/_uuidmodule.c

```diff
@@ -19,10 +19,16 @@ py_uuid_generate_time_safe(PyObject *Py_UNUSED(context),
 
     res = uuid_generate_time_safe(uuid);
     return Py_BuildValue("y#i", (const char *) uuid, sizeof(uuid), res);
-#elif HAVE_UUID_CREATE
+#elif defined(HAVE_UUID_CREATE)
     uint32_t status;
     uuid_create(&uuid, &status);
+# if defined(HAVE_UUID_ENC_BE)
+    unsigned char buf[sizeof(uuid)];
```

Comment:
The size should be 16 bytes.
https://www.freebsd.org/cgi/man.cgi?query=uuid_create
"must be large enough to hold the 16-octet binary UUID"

Reply:
Since @ned-deily wants this fix, I took the liberty of fixing this issue myself.

Reply:
Wait. My change only modified the size of the buffer, but not configure, not the call to Py_BuildValue two lines below. I reverted my change.

Anyway, the module constructor has an assertion for sizeof(uuid_t) == 16.